src/popup.css

```diff
@@ -0,0 +1,68 @@
+body {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+}
+
+.Box {
+  border: none;
+  box-shadow: none;
+}
+
+.Box-title {
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.form-checkbox {
+  margin-bottom: 12px;
+}
+
+.form-checkbox:last-child {
+  margin-bottom: 0;
+}
+
+.note {
+  font-size: 12px;
+  margin-top: 4px;
+}
+
+/* Slider styling for WebKit browsers */
+input[type='range']::-webkit-slider-thumb {
```

Comment:
Are these styling still required? `slider` sounds outdated word.

And keep the CSS minimum as possible as we do. And primer.css is still required? This is just a question. If you think keeping primer would be better, you can keep it.